bin/cam.js

```diff
@@ -1,14 +1,46 @@
-const cam = require('../lib/camera')
+// const scanner = require('../lib/scanner')
```

Comment:
Let's create a new script called mock-cam.js for this. It would be helpful to run the script on a live cam feed, as well, to test card capture thresholds.